data/h3_data_importer/Makefile

```diff
@@ -23,13 +23,13 @@ export AWS_ACCESS_KEY_ID = $(DATA_S3_ACCESS_KEY)
 export AWS_SECRET_ACCESS_KEY = $(DATA_S3_SECRET_KEY)
 
 all:
-	@aws s3 ls $(AWS_S3_BUCKET_URL) 2>&1 > /dev/null; \
-	if [ $$? -ne 0 ]; \
-	then \
-		echo "Error: Cannot access the expected AWS S3 bucket."; \
-		false; \
-	fi
-	@echo "AWS S3 bucket access confirmed, proceeding with import..."
+@aws s3 ls $(AWS_S3_BUCKET_URL) 2>&1 > /dev/null; \
+if [ $$? -ne 0 ]; \
+then \
+	echo "Error: Cannot access the expected AWS S3 bucket."; \
+	false; \
+fi
+@echo "AWS S3 bucket access confirmed, proceeding with import..."
```

Comment:
dunno if it runs ok with this change of identation??

Reply:
changed to the same indent that we had before.